Jantimizer/Histograms/DepthCalculators/DynamicDepth.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Histograms.DepthCalculators
+{
+    public class DynamicDepth

```

Comment:
Im not entirely sure why you have made this into some other classes that then attach to the managers.
Why have you not made a new histogram manager, called something like `DynDepthHistogramManager` instead? Because are we not not starting to mix what is equi depth, min depth and the new dynamic depth? Just seem rather overly confusing.